### PR TITLE
added getTokens function that extracts tokens from tokens.json config…

### PR DESCRIPTION
**Pull request by brandonfujii**
… file. Also, checks for error if file not found, or if there are errors in formatting.

### CODE REVIEW

**Review by thehandsomepanther — COMMENTED** [review]
have you tested this with team-switching (e.g. ctrl-T)? the tokens variable in index.js is just an array that gets looped over so getTokens() should return an array, but the format you have looks like it's just an object

File: package.json [hunk 1]
```diff
@@ -12,10 +12,12 @@
   "dependencies": {
     "blessed": "^0.1.81",
     "blessed-contrib": "^4.6.5",
+    "fs": "0.0.1-security",
     "jsonfile": "^2.4.0",
     "lodash": "^4.17.3",
     "node-env-file": "^0.1.8",
```
Comment (thehandsomepanther) [inline package.json:18]: we can get rid of this dependency too

**Comment by brandonfujii** [comment]
[Object.values()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values) returns an array if passed an object. I've also tested team-switching and it appears to be working on my end.

**Comment by thehandsomepanther** [comment]
gotcha

**Pull request merged**